styles/layouts/_main-navigation.scss

```diff
@@ -131,19 +131,19 @@
   transition: 0.3s ease;
   -webkit-transition: 0.3s ease;
   z-index: 999 !important;
-  min-height: 84px;
+  min-height: $navbar-desktop-min-height;
```

Comment:
this part of code can be a problem (responsive) but it's not related with this issue, lets keep it for another one

Reply:
If it breaks ui let's resolve it now please

Reply:
this was introduce before this PR (maybe https://github.com/kodadot/nft-gallery/commit/3433b00ec2b1f1233afd0497998ef5d6ee3071de & changed in https://github.com/kodadot/nft-gallery/pull/5093)